src/Buildalyzer/Logging/EventProcessor.cs

```diff
@@ -100,7 +96,7 @@ private void ProjectStarted(object sender, ProjectStartedEventArgs e)
 
             // Get the TFM for this project
             // use an empty string if no target framework was found, for example in case of C++ projects with VS >= 2022
-            var tfm = propertiesAndItems?.Properties.TryGet("TargetFrameworkMoniker")?.StringValue
+            string tfm = propertiesAndItems?.Properties.TryGet("TargetFrameworkMoniker")?.StringValue

```

Comment:
I thought `var` was okay now? Or is it this particular line where you want `string` instead?

Reply:
var is ok, I removed the warning in the config editor too.